src/core/server/models/user/validFeatureFlagsFilter.ts

```diff
@@ -0,0 +1,45 @@
+import { GQLFEATURE_FLAG } from "coral-server/graph/schema/__generated__/types";
```

Comment:
The `context` of this is the settings/feature flags, and not the user. This should instead be located in the settings folder.

Reply:
yeah wasn't sure, can do.